filament/src/materials/vsmMipmap.mat

```diff
@@ -0,0 +1,40 @@
+material {
+    name : vsmMipmap,
+    parameters : [
+        {
+            type : sampler2dArray,
+            name : color,
+            precision: high
+        },
+        {
+            type : int,
+            name : layer
+        },
+        {
+            type : int,
+            name : level
+        }
+    ],
+    outputs : [
+        {
+            name : color,
+            target : color,
+            type : float4
+        }
+    ],
+    domain : postprocess,
+    depthWrite : false,
+    depthCulling : false,
+    culling: none
+}
+
+fragment {
+    void postProcess(inout PostProcessInputs postProcess) {
+        highp vec2 uv = gl_FragCoord.xy * 2.0f /
+            vec2(textureSize(materialParams_color, materialParams.level).xy);
```

Comment:
How easy would it be to just pass the texture size as a uniform? Not a big deal but on some GPUs it's equivalent to a texture read.

Reply:
Ans if you did that, you could also avoid the divide -- you'd pass 2/size(level) in the uniform.

Reply:
Sure, I can do that